packages/cli/src/commands/generate/sdl/__tests__/fixtures/singleWordSdl.js

```diff
@@ -6,7 +6,7 @@ export const schema = gql`
     name: String
     email: String!
     isAdmin: Boolean!
-    profiles: UserProfile[]
+    profiles: [UserProfile]
```

Comment:
This means profiles is a nullable list of nullable `UserProfile` items. I'm guessing the list itself is not nullable and at least you get an empty array. (`[UserProfile]!`).

Reply:
From this [reference](https://github.com/prisma/prisma/issues/847) I think you are correct. I am however a bit confused to the `field.isRequired`, since `[UserProfile!]!' is also valid, but what does the property refer to?

I added the `!` to the `profiles` field.